packages/fragment-workflows/workflows-smoke-artifacts/scenario-matrix.js

```
@@ -0,0 +1,250 @@
+      );
+      if (over.length) {
+        findings.push(
+          `parallel ${id} exceeded attempts on ${over.map((s) => s.stepKey).join(",")}`,
+        );
+      }
+    }
+  }
+
+  if (findings.length) {
+    console.error(`Scenario matrix detected ${findings.length} anomalies`);
+    findings.slice(0, 10).forEach((finding) => console.error(`- ${finding}`));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)